inc/elementor/sections/background-color-classes.php

```diff
@@ -80,7 +80,7 @@ public function register_section( Controls_Stack $element, $section_id ) {
 					'{{WRAPPER}}' => '--ang_background_light_background: {{VALUE}};',
 					'{{WRAPPER}} .sk-light-bg:not(.elementor-column)' => 'background-color: {{VALUE}};',
 					'{{WRAPPER}} .sk-dark-bg .elementor-counter-title, {{WRAPPER}} .sk-dark-bg .elementor-counter-number-wrapper' => 'color: currentColor',
-					'{{WRAPPER}} .sk-light-bg.elementor-column > .elementor-element-populated' => 'background-color: {{VALUE}}; color: var(--ang_color_text_light)',
+					'{{WRAPPER}} .sk-light-bg.elementor-column > .elementor-element-populated' => 'background-color: {{VALUE}};',
```

Comment:
I see you're removing the text color, but we're Text Light color applied on background color classes, with these removed how'd we apply Background Color Classes > Text color?

Reply:
@gvgvgvijayan

Reply:
> I see you're removing the text color, but we're Text Light color applied on background color classes, with these removed how'd we apply Background Color Classes > Text color?

Added video message in slack product channel (https://analogwp.slack.com/archives/CJ72D4S03/p1605007628009200)